docs/lagoon-environments.md

```diff
@@ -84,3 +84,35 @@ needed to test certain functionalities. This can be achieved in two ways:
    ```sh
    lagoon config default --lagoon [instance name]
    ```
+
+### Using cron in pull request environments
+
+The `.lagoon.yml` has an environments section where it is possible to control various settings.
+On root level you specify the environment you want to address (eg.: `main`). And on the sub level of that you can define the cron settings.
```

Comment:
```suggestion
On root level you specify the environment you want to address (eg.: `main`). [On the sub level of that you can define the cron settings](https://docs.lagoon.sh/using-lagoon-the-basics/lagoon-yml/#environmentsnamecronjobs).
```

Having the key info first hand is good. Adding a reference to the documentation of cron jobs for bonus points.